pyproject.toml

```diff
@@ -1,31 +1,30 @@
 [tool.poetry]
 name = "plannerbenchmark"
-version = "0.2.0"
+version = "0.3.0"
 description = "Benchmark suite for local planner in dynamic environment."
 authors = ["Max Spahn <m.spahn@tudelft.nl>"]
 
 [tool.poetry.dependencies]
 python = "^3.6.2,<3.10"
-forwardkinematics = {git = "https://git@github.com/maxspahn/forwardKinematics.git", rev = "release-1.0"}
-planarenvs = {git = "https://git@github.com/maxspahn/gym_envs_planar.git", rev = "master"}
-urdfenvs = {git = "https://git@github.com/maxspahn/gym_envs_urdf.git", rev = "master"}
+planarenvs = "^1.0.0" 
+urdfenvs = "^0.2.2"
+motion-planning-scenes= "^0.1"
+forwardkinematics = "^1.0.1"
+fabrics = {path = "../../../../code/fabrics", optional = true}
 #robotmpcs = {git = "https://git@github.com/maxspahn/robot_mpcs.git", rev = "main", optional = true}
-motion-planning-scenes= {git = "https://git@github.com/maxspahn/motion_planning_scenes.git", rev = "main"}
-#fabrics = {path = "../../../../code/fabrics", optional = true}
-Sphinx = { version = "4.2.0", optional = true }
-sphinx-rtd-theme = { version = "1.0.0", optional = true }
-sphinxcontrib-napoleon = { version = "0.7", optional = true }
 
 
 [tool.poetry.extras]
 fabric = ['fabrics']
 mpc = ['robotmpcs']
-docs = ["Sphinx", "sphinx-rtd-theme", "sphinxcontrib-napoleon"]
 
 [tool.poetry.dev-dependencies]
 black = "^22.3.0"
 pytest = "^6.2"
 pylint = "^2.13.3"
+Sphinx = "4.2.0"
+sphinx-rtd-theme = "1.0.0"
+sphinxcontrib-napoleon = "0.7"
```

Comment:
That requires some changes in the gh-pages action. 
Installation should no longer include the `doc` option but the developer option.